gravity.sh

```diff
@@ -43,7 +45,7 @@ else
 fi
 
 #Remove the /* from the end of the IPv4addr.
-IPv4addr=${IPv4_address%/*}
+IPv4_address=${IPv4_address%/*}
```

Comment:
Future plans and things that are in work. Would prefer not having to undo all your hard work. I'll let the other members of the team comment as I may be misunderstanding. (Premature Optimization and all that...)